content/blogs/2023-10-09-why-ingestion-will-never-be-solved.md

```diff
@@ -0,0 +1,155 @@
+- How are plugins packaged and distributed? Are those built as binaries, `pip` packages or as Docker images?
+
+---
+
+## The differences between open-source data ingestion tools
+
+There are several significant differences between [Airbyte](https://airbyte.com/), [CloudQuery](https://www.cloudquery.io/) and [dltHub](https://dlthub.com/) based on how each tool approached those questions.
+
+### Airbyte
+
+[Airbyte](https://airbyte.com/) users can build [custom connectors](https://docs.airbyte.com/integrations/custom-connectors/) in many ways:
+1. With an [SDK for your preferred programming language](https://docs.airbyte.com/connector-development/cdk-python/) that autogenerates up to 75% of connector code based on provided API endpoints
+2. With a [low-code connector builder UI](https://docs.airbyte.com/connector-development/config-based/low-code-cdk-overview)
+3. Building a custom Docker image with connector code written in any language, and adding the connector metadata to [the Airbyte UI](https://docs.airbyte.com/integrations/custom-connectors/#adding-your-connectors-in-the-ui).
+
+Airbyte's connectors are packages as Docker images and executed as Docker containers. Airbyte itself is versioned using [semantic versioning](https://semver.org/), and connectors are versioned based on the Docker image tag.
+The connectors are maintained in a [monorepo](https://github.com/airbytehq/airbyte), and the Airbyte team is responsible for validating and merging all Pull Requests. However, note that you can build private custom connectors that don't have to be merged to the main platform. Contributing connectors to the community is encouraged, but not required.
+
+Airbyte's monorepo allows to centrally govern the connectors and ensure that they follow standards and are kept up-to-date. One downside of this approach is that the connectors are tied to the platform — it's not possible to run Airbyte connectors independently of the platform. Also, installing Airbyte requires cloning a fairly large repository. You can't pick and choose only plugins that you need (as [kestra](https://github.com/kestra-io/kestra) allows that) — in Airbyte, you install the product with all connectors baked in, which has its pros and cons.
+
+::alert{type="info"}
+To see how you can orchestrate Airbyte connectors with [kestra](https://github.com/kestra-io/kestra), check our [Airbyte blueprints](https://demo.kestra.io/ui/blueprints/community?page=1&q=airbyte), and [the following Airbyte documentation page](https://docs.airbyte.com/operator-guides/using-kestra-plugin/).
+::
+
+![airbyte_kestra.gif](/blogs/2023-10-09-why-ingestion-will-never-be-solved/airbyte_kestra.gif)
+
+Another distinction compared to [Singer](https://www.singer.io/) is that Airbyte is primarily designed to be used from the UI, not via code. This brings us to [CloudQuery](https://www.cloudquery.io/) and [dltHub](https://dlthub.com/) which are code-centric and intended to be used by developers.
```

Comment:
Maybe add the terraform provider that fill the gap ?

Reply:
ty, added